src/reducers/org.js

```diff
@@ -1371,10 +1371,39 @@ function updatePlanningItemsWithRepeaters({
   timestamp,
 }) {
   indexedPlanningItemsWithRepeaters.forEach(([planningItem, planningItemIndex]) => {
+    const adjustedTimestamp = applyRepeater(planningItem.get('timestamp'), timestamp);
     state = state.setIn(
       ['headers', headerIndex, 'planningItems', planningItemIndex, 'timestamp'],
-      applyRepeater(planningItem.get('timestamp'), timestamp)
+      adjustedTimestamp
     );
+    switch (planningItem.get('type')) {
```

Comment:
I confirm this works and the new test does, too.

But, I ask myself: Why isn't the header part of the `planningItems` in the first place? I might be mistaken, but it could be that the fix here is more explicit than it needs to be and that there's a bug further up the stack trace.

Reply:
> Why isn't the header part of the planningItems in the first place?

The active timestamps in title and description are added to `planningItems` on parse. Since there can be an arbitrary amount of timestamps it makes sense not to have one `planningItem` representing the title or the description. We need to preserve the place of a timestamp in title/description and we want to have it in a list of `planningItems`. So they necessarily exist in more than one place. There might be a cleaner solution where we store the timestamp only in one place and use references to that place but I don't see any extra benefit for what would be no negligible refactoring effort.

Scheduled / deadline timestamps are part of `rawDescription` but not of the parsed description. These timestamps only exist in one place so changing them there is visible and will be persisted.

Reply:
This is all very much correct and a perfect explanation. Thank you :pray: 

I'll add this as a comment to the code, because it's well written and quite informative.